wordpress-mobile/WordPress-Android
# [Site Creation] Step 1: Remove header and search input segments

This PR removes verticals header and search input segments as part of https://github.com/wordpress-mobile/WordPress-Android/issues/11420 and https://github.com/wordpress-mobile/WordPress-Android/pull/11421 and is one of multiple steps to remove verticals step (Step 1 of 3) from site creation flow. 

PR in FluxC (to remove `SegmentPromptModel`) - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1510. 

![device-2020-03-12-121042](https://user-images.githubusercontent.com/1405144/76494362-ad90be80-645a-11ea-9406-99c2cf1547f0.png)

**To test:**

(As parts of code using the FluxC `SegmentPromptModel` are removed from WPAndroid, we do not need to replace FluxC commit for https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1510 in WPAndroid, but it might be useful to do it while testing to check it doesn't cause any other issues. )

- Go to My site
- Click Switch Site 
- Click "+" button to launch Add new site 
- Click Create WordPress.com site on Add new site 
- Choose a segment Blog/Business/Professional/Blank Canvas to open Step 1 of 3 
- Verify segments for header and search input are not displayed

![device-2020-03-12-120821](https://user-images.githubusercontent.com/1405144/76494129-3d823880-645a-11ea-8326-387d1f798886.png)


PR submission checklist:
